src/system/AdminModule/Controller/AdminInterfaceController.php

```diff
@@ -0,0 +1,773 @@
+    					$menuitem['icon'] = 'info';
+    				} elseif ($menuitem['class'] == 'z-icon-es-locale') {
+    					$menuitem['class'] = null;
+    					$menuitem['icon'] = 'globe';
+    				} elseif ($menuitem['class'] == 'z-icon-es-locked') {
+    					$menuitem['class'] = null;
+    					$menuitem['icon'] = 'lock';
+    				} elseif ($menuitem['class'] == 'z-icon-es-log') {
+    					$menuitem['class'] = null;
+    					$menuitem['icon'] = 'archive';
+    				} elseif ($menuitem['class'] == 'z-icon-es-mail') {
+    					$menuitem['class'] = null;
+    					$menuitem['icon'] = 'inbox';
+    				} elseif ($menuitem['class'] == 'z-icon-es-new') {
+    					$menuitem['class'] = null;
+    					$menuitem['icon'] = 'file-o';
+    				} elseif ($menuitem['class'] == 'z-icon-es-ok') {
+    					$menuitem['class'] = null;
+    					$menuitem['icon'] = 'check';
+    				} elseif ($menuitem['class'] == 'z-icon-es-options') {
+    					$menuitem['class'] = null;
+    					$menuitem['icon'] = 'th-list';
+    				} elseif ($menuitem['class'] == 'z-icon-es-preview') {
+    					$menuitem['class'] = null;
+    					$menuitem['icon'] = 'cog';
+    				} elseif ($menuitem['class'] == 'z-icon-es-print') {
+    					$menuitem['class'] = null;
+    					$menuitem['icon'] = 'print';
+    				} elseif ($menuitem['class'] == 'z-icon-es-profile') {
+    					$menuitem['class'] = null;
+    					$menuitem['icon'] = 'user';
+    				} elseif ($menuitem['class'] == 'z-icon-es-regenerate') {
+    					$menuitem['class'] = null;
+    					$menuitem['icon'] = 'refresh';
+    				} elseif ($menuitem['class'] == 'z-icon-es-remove') {
+    					$menuitem['class'] = null;
+    					$menuitem['icon'] = 'times';
+    				} elseif ($menuitem['class'] == 'z-icon-es-save') {
+    					$menuitem['class'] = null;
+    					$menuitem['icon'] = 'floppy-o';
+    				} elseif ($menuitem['class'] == 'z-icon-es-saveas') {
+    					$menuitem['class'] = null;
+    					$menuitem['icon'] = 'floppy-o';
+    				} elseif ($menuitem['class'] == 'z-icon-es-search') {
+    					$menuitem['class'] = null;
+    					$menuitem['icon'] = 'search';
+    				} elseif ($menuitem['class'] == 'z-icon-es-url') {
+    					$menuitem['class'] = null;
+    					$menuitem['icon'] = 'globe';
+    				} elseif ($menuitem['class'] == 'z-icon-es-user') {
+    					$menuitem['class'] = null;
+    					$menuitem['icon'] = 'user';
+    				} elseif ($menuitem['class'] == 'z-icon-es-view') {
+    					$menuitem['class'] = null;
+    					$menuitem['icon'] = 'eye';
+    				} elseif ($menuitem['class'] == 'z-icon-es-warning') {
+    					$menuitem['class'] = null;
+    					$menuitem['icon'] = 'exclamation-triangle';
+    				} elseif ($menuitem['class'] == 'z-icon-es-rss') {
+    					$menuitem['class'] = null;
+    					$menuitem['icon'] = 'rss-square';
+    				}
+    			}
+    	
+    			$active = '';
+    			if (!empty($menuitem['url']) && System::getBaseUrl().$menuitem['url'] === System::getCurrentUrl()) {
```

Comment:
do not use static classes wherever possible. This `System` information can be obtained from the `Request` object. `$this->container->get('Request');`